daemon/endpoint.go

```diff
@@ -675,7 +676,7 @@ func (d *Daemon) UpdateSecLabels(id string, add, del labels.Labels) middleware.R
 				logfields.Identity:   identity.ID,
 			}).WithError(err).Warn("Unable to release temporary identity")
 		}
-		return NewPutEndpointIDLabelsNotFound()
+		return PutEndpointIDLabelsNotFoundCode, fmt.Errorf("")
```

Comment:
Why an empty error?

Reply:
Apparently because I omitted it ;) I'll fix this up.